builtin/tag.c

```diff
@@ -106,7 +106,28 @@ static int delete_tag(const char *name, const char *ref,
 static int verify_tag(const char *name, const char *ref,
 				const unsigned char *sha1)
 {
-	return gpg_verify_tag(sha1, name, GPG_VERIFY_VERBOSE);
+	int flags;
+	flags = GPG_VERIFY_VERBOSE;
+
+	if (fmt_pretty) {
+		//verify_ref_format(fmt_pretty);
+
+		// Create a `ref_array_item` in order to use `show_ref_array_item`
+		// XXX: Just copies code from `static new_ref_array_item` in ref_filter.c
+		struct ref_array_item *ref_item;
+		FLEX_ALLOC_STR(ref_item, refname, name);
```

Comment:
`refname` is the name of the variable in [`struct ref_array_item`] (https://github.com/lukpueh/git/blob/master/ref-filter.h#L42). See also the [`FLEX_ALLOC_STR` macro] (https://github.com/lukpueh/git/blob/master/git-compat-util.h#L850).